.github/workflows/build_deploy_rcb_fe.yml

```diff
@@ -0,0 +1,179 @@
+name: Build & Deploy RecipeBox Frontend
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - recipe-box-frontend/**
+      - .github/workflows/build_deploy_rcb_fe.yml
+  pull_request:
+    branches:
+      - main
+    paths:
+      - recipe-box-frontend/**
+      - .github/workflows/build_deploy_rcb_fe.yml
```

Comment:
Nu vrem sa facem deploy cand se creaza pr, dar o sa avem de adaugat check pe viitor doar cu build step, asta vom face in alt ticket

Reply:
asta e deja facut, e un if pus pe deploy